Yoast/javascript
# Rearrange folders related to tree research

Fixes #119 

## Summary
Renamed the `tree` folder to `parsedPaper`, because there will be an object of that name containing the tree and metadata. This required a lot of imports to be editted.

## Test instructions

1. `lerna bootstrap` in the root of your cloned javascript repo.
2. Go to `apps/content-analysis` and run `yarn start`. Verify that the tree building and analysis continue without error. (pro tip: test your go-to text on develop, verify the scores and tree, and then switch to this branch and do the same).
